main.py

```diff
@@ -877,7 +877,11 @@ def print_diff(
         header_to_remove: str,
         header_to_change: str,
     ) -> None:
```

Comment:
```suggestion
    ) -> bool:
```
We should update the type signature to indicate that the function returns a boolean now. You can typecheck the file with `mypy --strict`.